supyr_struct/field_type_methods/encoders.py

```diff
@@ -0,0 +1,198 @@
+'''
+Encoder functions for all standard FieldTypes.
+
+Encoders are responsible for converting a python object into bytes*
+
+*Not all encoders return bytes objects.
+FieldTypes that operate on the bit level cant be expected to return
+even byte sized amounts of bits, so they operate differently.
+A FieldTypes serializer and encoder simply need to
+be working with the same parameter and return data types.
+'''
+__all__ = [
+    # basic encoders
+    'encode_numeric', 'encode_string', 'no_encode',
+    'encode_big_int', 'encode_bit_int',
+    # specialized encoders
+    'encode_24bit_numeric', 'encode_decimal', 'encode_bit', 'encode_raw_string',
+    'encode_int_timestamp', 'encode_float_timestamp', 'encode_string_hex',
+
+    # wrapper functions
+    'encoder_wrapper', 
+    ]
+
+from decimal import Decimal
+from struct import pack
+from time import mktime, strptime
+
+from supyr_struct.defs.constants import ATTR_OFFS
+
+
+# This function is for wrapping encoders in functions which properly
+# work with FieldTypes where is_block and is_data are both True.
+# This is because the node will be a Block with some attribute
+# that stores the "data" of the node.
+def encoder_wrapper(en):
+    '''
+    '''
```

Comment:
Toss that comment into that docstring. :)

Reply:
kk